osu.Game/Skinning/LegacySkin.cs

```diff
@@ -571,13 +562,13 @@ private IEnumerable<string> getLegacyLookupNames(HitSampleInfo hitSample)
             yield return hitSample.Name;
         }
 
-        private IEnumerable<string> getFallbackNames(string componentName)
+        private IEnumerable<string> getFallbackSampleNames(string name)
```

Comment:
I do wonder if we can get rid of this crap too, eventually.

Reply:
Yeah, but that one is a bit more involved and has multiple usages that I'm not sure is worth spending my time on at this particular moment.